### ISSUE TITLE
Device Not Being Shown in The List

### ISSUE THREAD

**Issue by abidkhanpk** [madlymad/mac-scrcpy-app#7]
Hi,

It seems nice to have an application to run scrcpy instead of through terminal. I have download the dmg but unable to connect the device. Device display is obtained through terminal successfully but nothrough the app, it is not. It always say, no device found, connect an android device even though the device is connected through the USB.
Moreover, there is one more request/query, does it connect through the wifi or only through only usb? It would nice if it could work over wifi too.

Best Regards

**Comment by madlymad** [comment]
@PacManPk hey I just noticed your post, you can check this script https://github.com/madlymad/mac-scrcpy-app/blob/ea710b3b9e47e4621b300868f7a703d6802083bc/scripts/list_devices.sh to understand why the device is not found and what is in different location at your mac.

The command that do the device listing is:
```
adb devices | grep -w 'device' | cut -f1 | xargs -I X sh -c \
  'c=$(adb -s X shell getprop ro.product.manufacturer); m=$(adb -s X shell getprop ro.product.model); s=$(echo X); echo "$s ($c $m)"'
```

If you modify this exact script it would be possible to also work with Wi-Fi, simplest way would be to configure your network with a static IP for your phone, with a dynamic IP would be a bit complex as you would require some extra manual steps to find the device IP and do the connection etc etc, so it is not straightforward for me to write all that in scripts!
See official doc https://developer.android.com/tools/adb#wireless-android11-command-line

**Comment by abidkhanpk** [comment]
For wifi, can it not be modified in such a way that it asks user to inpur IP address on opening in wifi mode? It will be an improvement in this app.

**Comment by madlymad** [comment]
@PacManPk so the reason that you don't see the device is that you try to connect it via Wi-Fi?

WiFi connection is not in the intentions of this launcher cause it will create extra complexity and at least one extra click to pick the way you want to connect the device. The design of the launcher is a single click device screen share one device or simumaneous share two connected devices! 

You can use an external tool or command to connect your device for example type in the terminal `adb pair ipaddr:port` and then use the launcher for cause I assume that the command `adb devices` will list your device.
You can also add more scripts before these to pair your device you can go through my article that is in the readme https://github.com/madlymad/mac-scrcpy-app?tab=readme-ov-file#manual and understand the logic to extend it. 

Personally, I don't have such a device so I am unable to test or provide further support on WiFi connecting and I am far from messing with an endless troubleshooting of adb connectivity issues.

**Comment by abidkhanpk** [comment]
No, I don't connected it through wifi. I intend to but after I successfully get it connected through USB which, unfortunately, I'm still unable to.
